app/src/renderer/js/editor.js

```diff
@@ -81,6 +84,21 @@ document.addEventListener('DOMContentLoaded', () => {
     preview.play();
   }
 
+  function skip(event) {
+    const rect = event.target.getBoundingClientRect();
+    const x = event.pageX - rect.left; // X position within the trimmer
+    const skipTime = preview.duration * (x / rect.width); // Calculated time in seconds where the click happened
+
+    if (skipTime < getTrimmerValue(trimmerIn)) {
+      return;
+    } // Clicked before the first trimmer
```

Comment:
Put the comment above the block and add an empty line between the if statements.